cfy_manager/main.py

```diff
@@ -775,6 +788,13 @@ def _configure_supervisord():
     sudo('systemctl restart supervisord', ignore_failures=True)
 
 
+def _create_packages_installed_file():
+    for service_name in config[SERVICES_TO_INSTALL]:
+        if service_name in MAIN_SERVICES_NAMES:
+            update_yaml_file(INSTALLED_PACKAGES,
+                             {service_name: _get_packages()})
```

Comment:
Same here, should not we have for each service only contains its relavent packages ? like:
```
database_service:
- postgresql95
- postgresql95-server
- postgresql95-contrib
- libxslt
- libestr
- libfastjson
- rsyslog
- etcd
- patroni

manager_service:
 - ...
```

unless we need for each service to make sure that other services already existsed in case cluster with 3 nodes as  each service should have all other services as well ?

Reply:
Each service will contain only its relevant packages. Let's take for example the use-case of a 3 nodes cluster. When we run `cfy_manager install -c /etc/cloudify/<postgresql-config.yaml>`, the `_get_packages()` function goes over the `services_to_install` section in the `<postgresql-config.yaml>` file and returns the relevant packages to the services listed in the `services_to_install` section (e.g. `database_service` and `monitoring_service`). Then, the packages.yamk will be updated only with the dictionary `{database_service: <list_of_relevant_packages>}`. Afterwards the same would happen to the rabbitmq and manager.